templates/generic/rpi4_reference_dunfell.json

```diff
@@ -94,12 +94,27 @@
                     "rw",
                     "X-dobby.optional"
                 ]
+            },
```

Comment:
This binds the whole /tmp from the host. That is too much. For the rialto socket add this:

            {
                "source": "/tmp/{id}",
                "destination": "/tmp/rialto-0",
                "type": "bind",
                "options": [
                    "rbind",
                    "rw",
                    "X-dobby.optional"
                ]
            }

Same as I do here: https://github.com/rdkcentral/BundleGen/blob/c6c2922d4889643ae69ada992c19126e310ecec7/templates/generic/rpi3_reference_dunfell.json#L89